gui.py

```diff
@@ -344,13 +356,44 @@ def _change_language(values: Dict, window: Sg.Window) -> None:
     _update_text_lang(window)
 
 
+def _update_playlist_index_state(checked: bool, playlist_items: str, window: Sg.Window):
```

Comment:
It would be better if there is a checkbox for the Indices field which is only enabled if Playlist is ticked. That way the use wouldn't be led to think he has to input indices and will understand it's only an option